contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -89,6 +101,89 @@ public VoldemortSwapJob(String id,
         this.failedFetchStrategyList = failedFetchStrategyList;
         this.clusterName = clusterName;
         this.buildPrimaryReplicasOnly = buildPrimaryReplicasOnly;
+        this.props = props;
+    }
+
+    private void runDistcp(Path from, Path to) throws Exception {
+        info("     source: " + from);
+        info("destination: " + to);
+        EmbeddedGobblin embeddedGobblin = new EmbeddedGobblinDistcp(from, to).mrMode();
+
+        // Used for global throttling"
+        embeddedGobblin.distributeJar("lib/*");
+
+        for (Map.Entry<String, String> entry : this.props.entrySet()) {
+            if (entry.getKey() instanceof String && ((String) entry.getKey()).startsWith("distcpConf.")) {
```

Comment:
This block of code is copied from embedded-gobblin-example. Let me take a closer look.